src/Microsoft.Health.Fhir.CosmosDb/Features/Health/CosmosStatusReporter.cs

```diff
@@ -0,0 +1,36 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Health.Core.Features.Health;
+using Microsoft.Health.Encryption.Customer.Health;
+using Microsoft.Health.Fhir.Api.Features.Health;
+using Microsoft.Health.Fhir.Core.Features;
+
+namespace Microsoft.Health.Fhir.CosmosDb.Features.Health
+{
+    /// <summary>
+    /// Cosmos DB implementation of <see cref="IDatabaseStatusReporter"/>.
+    /// Always returns healthy status without performing any checks.
+    /// </summary>
+    public class CosmosStatusReporter : IDatabaseStatusReporter
+    {
+        public CosmosStatusReporter()
```

Comment:
You can remove the constructor. If it's empty, then we can assume the default constructor.